src/index.js

```diff
@@ -46,6 +51,9 @@ exports.pushStream = async (
       case 'INSERT': {
         let body = converter(record.dynamodb.NewImage)
         body = removeEventData(body)
+        if (transformFunction) {
+          body = transformFunction(body)
```

Comment:
What do you think of passing `record.eventName` to this function?
`transformFunction(body, record.eventName)`
then you can manipulate based on `eventName`?

Reply:
Well, you could easily do that, but I don't see the value: Unless I'm missing something, you either delete a record, in which case you will use the id, or create/update a record, in which case you need to do the same data transformations.
Please let me know if you have a different point of view.